src/DynamoCoreWpf/Views/PackageManager/Controls/PackageManagerPublishControl.xaml

```diff
@@ -3,13 +3,13 @@
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-             xmlns:local="clr-namespace:Dynamo.PackageManager.UI"
              xmlns:controls="clr-namespace:Dynamo.Controls"
              xmlns:p="clr-namespace:Dynamo.Wpf.Properties"
              xmlns:ui="clr-namespace:Dynamo.UI"
-             xmlns:fa="http://schemas.fontawesome.io/icons/"
              xmlns:packagemanager="clr-namespace:Dynamo.PackageManager"
+             xmlns:xctk="http://schemas.xceed.com/wpf/xaml/toolkit"
```

Comment:
@QilongTang please recall that we are using a third party fork of the real toolkit as their license changed - and I would advise not adding more dependencies on this fork if possible.

Reply:
That does make sense, but @dnenov is there a different way to implement a spinner, I guess all WPF spinner depend on some kind of UI toolkit?

Reply:
We can then create our own control, which will be harder but of course, can be done using native WPF elements. I will get on with it, I will reuse this PR and make it a draft one for the time being.

Reply:
I just published the rework with native wpf custom user control. This removes the dependency on xceed for this project. Marking it as 'ready for review'